rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -1637,6 +1639,20 @@ void main_window::SetIconSizeActions(int idx)
 		ui->setIconSizeLargeAct->setChecked(true);
 }
 
+void main_window::RemoveDiskCache()
+{
+	std::string cacheDir = Emulator::GetHdd1Dir();
```

Comment:
cache in system.cpp is
`const std::string cache_location = Emulator::GetHdd1Dir() + "/cache";`
Why the change here?

Reply:
Cache folder is easier to iterate through for the limiter. This one just deletes everything so I might as well just delete absolutely everything.

Most data is stored in the cache folder but I've seen some dangling files get left to the root of the drive or other folders in it. I'm not sure what the logic is there.

Reply:
Changed as requested